chall02/edramire.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import sys
+import re
+
+morse = {
+	"A": ".-",
+	"B": "-...",
+	"C": "-.-.",
+	"D": "-..",
+	"E": ".",
+	"F": "..-.",
+	"G": "--.",
+	"H": "....",
+	"I": "..",
+	"J": ".---",
+	"K": "-.-",
+	"L": ".-..",
+	"M": "--",
+	"N": "-.",
+	"O": "---",
+	"P": ".--.",
+	"Q": "--.-",
+	"R": ".-.",
+	"S": "...",
+	"T": "-",
+	"U": "..-",
+	"V": "...-",
+	"W": ".--",
+	"X": "-..-",
+	"Y": "-.--",
+	"Z": "--..",
+	" ": " "
+}
+msg = "usage: ./edramire.py <a-zA-Z string>"
+argn = len(sys.argv)
+if argn != 2:
+	print(msg)
+else:
+	string = sys.argv[1]
+	str_find = re.search(r'[^A-Za-z ]', string)
```

Comment:
You could use `isascii` or `isalpha` instead of a regexp, or if you're afraid to find a character that is not allowed, check if the characters received are in your dictionary